### DISCUSSION TITLE
Custom element doesn't exist:alarm-keypad-card

### DISCUSSION THREAD

**Discussion by olddawgpowers** [Dilbert66/esphome-dsckeybus#85] [Q&A]
For some reason my alarm-keypad-card no longer works?  I am running HA 2023.5.2.    I am not sure when it quit working because I seldom use the keypad and it is on its own page.   I have tried clearing/refreshing the browser and changing id in both the resources and yaml?
![image](https://user-images.githubusercontent.com/36567590/236709632-82392135-b4c9-43c0-af7d-523fa1d75c43.png)

![image](https://user-images.githubusercontent.com/36567590/236709584-90961ffd-423d-496a-91e5-93fe86977b8d.png)

**Comment by Dilbert66** [comment]
Get the latest panel card script from the dev branch. It was updated a while ago to work with the latest HA.

**Reply by olddawgpowers** [reply]
Thanks.  That did it.  I noticed I also put the wrong name in the resource when I was trying to fix it.
